src/components/footer/footer.js

```diff
@@ -1,84 +1,104 @@
 import React from 'react';
 import preval from 'preval.macro';
 import moment from 'moment';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
 
 const buildTimestamp = preval`module.exports = new Date().getTime();`;
 
 const Footer = () => {
   return (
-    <footer className="bg-black py-6 md:py-4 lg:py-2">
-      <section className="main-container">
-        <div className="xl:mx-6">
-          <div className="mx-6 flex flex-col-reverse md:flex-row justify-between">
-            <div className="py-2">
-              Copyright © SSW 1990 - {new Date().getFullYear()}. All Rights
-              Reserved.
+    <>
+      <div
+        className="py-2 md:py-4 text-center bg-grey-translucent"
+        style={{ fontSize: '.9rem' }}
```

Comment:
I'd avoid style tags where possible.
.text-sm gives a font-size of .875rem which is pretty close to what you are trying to achieve

Reply:
done. Thanks for the advice.